pages/k8s/how-to-containers.md

```diff
@@ -0,0 +1,32 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Container runtimes"
+  description: Using container runtimes in Charmed Kubernetes
+keywords: containers, runtimes
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: how-to-containers.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+In addition to the standard runtime, Charmed Kubernetes supports a variety of container runtime options. For more specific information, please choose from the topics below:
+
+- [Containerd & Docker](/kubernetes/docs/container-runtime)
```

Comment:
We haven't supported Docker since 1.23 so I would be tempted to drop it from the link text here.